src/opnsense/mvc/app/models/OPNsense/Proxy/Proxy.xml

```diff
@@ -338,9 +338,15 @@
                 <default>1</default>
                 <Required>Y</Required>
             </addACLforInterfaceSubnets>
-            <transparentMode type="BooleanField">
-                <default>0</default>
+            <transparentMode type="OptionField">
+                <default>off</default>
                 <Required>Y</Required>
+                <BlankDesc>Default</BlankDesc>
+                <OptionValues>
+                    <off>Turn off transparent Mode</off>
+                    <bump>Do SSL-Bump in transparent Mode</bump>
+                    <splice>Log SNI only in transparent Mode</splice>
+                </OptionValues>
```

Comment:
we should try to avoid the model bump or at least "smart" reuse of the same setting plus migration altogether. changing existing option values and their behaviour at the same time is a recipe for disaster (config restore on older versions, testing and migration time spent in the hopes it'll not break which it ultimately will)